test/test_util.js

```diff
@@ -360,3 +362,26 @@ async function addKeyHashToMap (k8, nodeId, keyDir, uniqueNodeDestDir, keyHashMa
   const keyString = keyBytes.toString()
   keyHashMap.set(privateKeyFileName, crypto.createHash('sha256').update(keyString).digest('base64'))
 }
+
+/**
+ * @param {ConfigManager} configManager
+ * @returns {K8}
+ */
+export function getK8Instance (configManager) {
+  try {
+    return new K8(configManager, testLogger)
```

Comment:
so, on the pipeline this command runs and passes and allows the test cases to execute without interacting with a live cluster, but when my machine it would fail if kube context is not set to an active cluster. 

I see below that you have added code to create a kind cluster, but, the goal was to some how possibly load a mock or something or update the code so that it would not fail when a kube config is found but the kubernetes cluster is not running, since it isn't really needed for the unit test case.  

Also, we don't want to bind ourselves to Kind.

Reply:
```suggestion
    return new K8(configManager, testLogger)
    // TODO: return a mock without running the init within constructor after we convert to Mocha, Jest ESModule mocks are broke.
```